Adyen/UserInterface/01a_Connections.md

```diff
@@ -0,0 +1,108 @@
+
+The CONNECTIONS tab provides an overview of all connections created for *Payments*. It also allows you to configure these connections.   
+For detailed information on general UI functions and procedures of the *CONNECTIONS* tab, see [Connections](../../Payments/UserInterface/08a_Connections.md) in the *Payments* documentation.
+
+
+## Edit Adyen connection &ndash; Credentials
+
+*Payments > Settings > Tab CONNECTIONS > Select Adyen&trade; connection > Tab Credentials*
+
+![Edit Adyen credentials](../../Assets/Screenshots/Adyen/Integration/EditCredentials.png "[Edit Adyen credentials]")
+
+- *Driver*  
+    Selected driver name. This drop-down list is read-only.
+
+**Credentials**
+
+The *Credentials* tab displays the connection credentials that you have defined in Adyen&trade;. This is a copy of those credentials. This means that the credentials in Adyen&trade; and the *Payments* module must be the same. 
+
+- *Checkout API endpoint*   
+   Endpoint of the Checkout API that is managed in Adyen&trade;. For detailed information on the Checkout API, see [Adyen Checkout API](https://docs.adyen.com/api-explorer/Checkout/71/overview "[https://docs.adyen.com/api-explorer/Checkout/71/overview]") in the Adyen&trade; documentation. For the test environment it is `https://checkout-test.adyen.com/checkout/`, for the live account you get the endpoint directly from Adyen&trade;. 
+     
+- *Checkout API key*    
+   Key of the *Checkout API* endpoint that is managed in the Adyen&trade; backend.
+
+- *Payment API endpoint*   
+   Endpoint of the *Payment API* that is managed in Adyen&trade;. For the test environment, it is `https://pal-test.adyen.com/pal/servlet/Payment/`, for the live account you get the endpoint directly from Adyen&trade;.
+
+- *Payment user*   
+   Username of the technical user that is managed in the Adyen&trade; backend.
+
+- *Payment password*   
+   Password of the *Checkout* API that is managed in the Adyen&trade; backend.
+
+ - *Report user*  
+     Username of the *Report API* that is managed in the Adyen&trade; backend.
+
+ - *Report password*    
+    Password of the *Report API* you have created in the settings of the Adyen&trade; backend.
+
+- *Notification user*  
+    Edit the username of the technical user. You can edit the notification user both in Actindo or in Adyen&trade;. If you do any changes here, remember to follow these changes in the Adyen&trade; webhook configuration. For detailed information, see [Create Adyen webhook](../Integration/01_ManageAdyenSettings.md#create-adyen-webhook).
+
+- *Notification password*   
+   Edit the password of the technical user. If you do any changes here, remember to follow these changes in the Adyen&trade; webhook configuration.
+
+- *Merchant account*   
+  Edit the merchant account of the connection you are currently editing.
+
+- ![Toggle](../../Assets/Icons/Toggle.png "[Toggle]") *MIT fraud check*   
+   Enable this toggle if you want to do your own fraud check.
```

Comment:
not quite, this toggle is there to disable the adyen fraud checks for MIT transactions. this is advisable if we get a lot of false positive fraud flags when trying to process MIT requests. as the fraud check is usually done when the payment cresentials are saved in adyen when for the alter mit processing already, this is usually save to skip

Reply:
Done